<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmKeyBoard</class>
 <widget class="QWidget" name="SmKeyBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	font: 75 15px &quot;Arial&quot;;
	font-weight:bold;
	background-color:rgba(251,219,167,0);
}
/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] #widget
{
	background-color:rgba(255,255,255,1);
	border:1px solid gray;
}
#SmKeyBoard[pltdismode =&quot;indoor&quot;] #pb_cancel
{
	background-color:transparent;
	border-image: url(:/bit/image/keyBoard/indoor/btn_close_w30_normal.png);
}
#SmKeyBoard[pltdismode =&quot;indoor&quot;] #pb_cancel:pressed
{
	background-color:transparent;
	border-image: url(:/bit/image/keyBoard/indoor/btn_close_w30_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] #widget
{
	background-color:rgba(255,255,255,1);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] #pb_cancel
{
	background-color:transparent;
	border-image: url(:/bit/image/keyBoard/indoor/btn_close_w30_normal.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] #pb_cancel:pressed
{
	background-color:transparent;
	border-image: url(:/bit/image/keyBoard/indoor/btn_close_w30_click.png);
}

/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] #widget
{
	background-color:rgba(251,219,167,1);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] #pb_cancel
{
	background-color:transparent;
	border-image: url(:/bit/image/keyBoard/night/btn_close_normal.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] #pb_cancel:pressed
{
	background-color:transparent;
	border-image:url(:/bit/image/keyBoard/night/btn_close_click.png) ;
}

</string>
  </property>
  <property name="pltdismode" stdset="0">
   <string>indoor</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>440</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:rgba(0,0,0,0);
}</string>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>196</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>233</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_7_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_7_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_7_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_7_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/7.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/7-click.png);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_1_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_1_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_1_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_1_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/1.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/1-click.png);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_8">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>233</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_8_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_8_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_8_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_8_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/8.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/8-click.png);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_dot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>287</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_dot_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image:url(:/bit/image/keyBoard/indoor/btn_dot_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_dot_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image:url(:/bit/image/keyBoard/indoor/btn_dot_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/dot.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/dot-click.png);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_add">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>287</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_add_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image:url(:/bit/image/keyBoard/indoor/btn_add_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_add_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image:url(:/bit/image/keyBoard/indoor/btn_add_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/add.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/add-click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_3">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>125</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_3_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_3_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_3_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_3_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/3.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/3-click.png);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>179</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_4_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_4_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_4_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_4_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/4.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/4-click.png);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image:url(:/bit/image/keyBoard/indoor/btn_clear_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_clear_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image:url(:/bit/image/keyBoard/indoor/btn_clear_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_clear_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image:url(:/bit/image/keyBoard/night/Clear.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image:url(:/bit/image/keyBoard/night/Clear-click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_enter">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>367</y>
      <width>55</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
f	ont-weight:bold;
	font-family:&quot;Arial&quot;;
	color:white;
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_w55_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image:url(:/bit/image/keyBoard/indoor/btn_w55_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	font-weight:bold;
	font-family:&quot;Arial&quot;;
	color:white;
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_w55_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image:url(:/bit/image/keyBoard/indoor/btn_w55_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	font-weight:bold;
	font-family:&quot;Arial&quot;;
	color:rgba(252, 220, 168, 1);
	background-color:rgba(0,0,0,0);
	border-image:url(:/bit/image/keyBoard/night/btn_w70_normal.png) ;
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/btn_w70_click.png);
}</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_2">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>125</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_2_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_2_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_2_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_2_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/2.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/2-click.png);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_del">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>70</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_del_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_del_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_del_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_del_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image:url(:/bit/image/keyBoard/night/delete.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image:url(:/bit/image/keyBoard/night/delete-click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_6">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>179</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_6_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_6_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_6_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_6_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/6.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/6-click.png);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_0">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>287</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode = &quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_0_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_0_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_0_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_0_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/0.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/0-click.png);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_5">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>179</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_5_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_5_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_5_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_5_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/5.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/5-click.png);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_9">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>233</y>
      <width>74</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*************************indoor*********************/
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_9_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;indoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_9_w74_click.png);
}
/*************************outdoor*********************/
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/indoor/btn_9_w74_nor.png);
}
#SmKeyBoard[pltdismode=&quot;outdoor&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/indoor/btn_9_w74_click.png);
}
/*************************night*********************/
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton
{
	background-color:rgba(0,0,0,0);
	border-image: url(:/bit/image/keyBoard/night/9.png);
}
#SmKeyBoard[pltdismode=&quot;night&quot;] QPushButton:pressed
{
	border-image: url(:/bit/image/keyBoard/night/9-click.png);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>15</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
